cellpack/autopack/loaders/migrate_v2_to_v2_1.py

```diff
@@ -0,0 +1,39 @@
+import copy
+
+
+def convert_partners(object_data):
```

Comment:
This is great to have sequential conversions!

Reply:
agree!